src/PKSim.CLI/Program.cs

```diff
@@ -35,22 +40,35 @@ static int Main(string[] args)
 
       private static void startSnapshotsRun(SnapshotRunCommand snapshotRunOptions)
       {
-         startCommand<SnapshotRunner, SnapshotRunOptions>("Snapshot run", snapshotRunOptions.ToRunOptions());
+         startCommand<SnapshotRunner, SnapshotRunOptions>("Snapshot run", snapshotRunOptions);
       }
 
       private static void startJsonRun(JsonRunCommand jsonRunOptions)
       {
-         startCommand<JsonSimulationRunner, JsonRunOptions>("Batch run", jsonRunOptions.ToRunOptions());
+         startCommand<JsonSimulationRunner, JsonRunOptions>("Batch run", jsonRunOptions);
       }
 
-      private static void startCommand<TBatchRunner, TStartOptions>(string command, TStartOptions startOptions) where TBatchRunner : IBatchRunner<TStartOptions>
+      private static void startCommand<TBatchRunner, TRunOptions>(string command, RunCommand<TRunOptions> runCommand) where TBatchRunner : IBatchRunner<TRunOptions>
       {
-         Console.WriteLine($"Starting {command.ToLowerInvariant()} with arguments:");
-         Console.WriteLine(startOptions.ToString());
+         var logger = initializeLogger(runCommand);
+         logger.AddInfo($"Starting {command} with arguments:\n{runCommand}");
          ApplicationStartup.Start();
          var runner = IoC.Resolve<TBatchRunner>();
-         runner.RunBatchAsync(startOptions).Wait();
-         Console.WriteLine($"{command} finished");
+         runner.RunBatchAsync(runCommand.ToRunOptions()).Wait();
+         logger.AddInfo($"{command} finished");
+      }
+
+      private static ILogger initializeLogger(RunCommand runCommand)
+      {
+         var loggerFactory = IoC.Resolve<ILoggerFactory>();
+     
+         loggerFactory
```

Comment:
This is how you finally call it: The CLI wants console log and File log. Done once and that's it.